docs/_partials/page.hbs

```diff
@@ -82,6 +90,28 @@
       <script src="{{relative '/dist/docs.js'}}"></script>
       <script> window.intercomSettings = { app_id: 'i82zr1ej' }</script>
       <script>(function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/i82zr1ej';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);}if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})()</script>
+      <!-- TAG COMMANDER START //-->
+      <script type="text/javascript">
+      //<![CDATA[
+          var tc_vars = {
+            env_work: 'prod',
+            page_name: '/toolkit',
```

Comment:
Isn't this supposed to contain the URL of the current page, instead of containing `/toolkit` all the time?
You could use the `link` property with the `BASE_URL` env variable for that.